app/src/main/res/values/strings.xml

```diff
@@ -119,105 +119,105 @@
+    <string name="title_seekbar_deadzone">Analog stick deadzone adjustment</string>
     <string name="suffix_seekbar_deadzone">%</string>
     <string name="title_checkbox_xb1_driver">Xbox 360/One USB gamepad driver</string>
-    <string name="summary_checkbox_xb1_driver">Enables a built-in USB driver for devices without native Xbox controller support</string>
-    <string name="title_checkbox_usb_bind_all">Override native Xbox gamepad support</string>
-    <string name="summary_checkbox_usb_bind_all">Use Moonlight\'s USB driver for all supported gamepads, even if native Xbox controller support is present</string>
-    <string name="title_checkbox_mouse_emulation">Mouse emulation via gamepad</string>
-    <string name="summary_checkbox_mouse_emulation">Long pressing the Start button will switch the gamepad into mouse mode</string>
-    <string name="title_checkbox_mouse_nav_buttons">Enable back and forward mouse buttons</string>
-    <string name="summary_checkbox_mouse_nav_buttons">Enabling this option may break right clicking on some buggy devices</string>
+    <string name="summary_checkbox_xb1_driver">Uses a built-in driver on devices without Xbox controller support</string>
+    <string name="title_checkbox_usb_bind_all">Override Xbox gamepad support</string>
+    <string name="summary_checkbox_usb_bind_all">Use Moonlight\'s driver for all supported gamepads, even if Xbox controllers are supported</string>
+    <string name="title_checkbox_mouse_emulation">Mouse via gamepad</string>
+    <string name="summary_checkbox_mouse_emulation">Long pressing the \'Start'\ button switches the gamepad into mouse mode</string>
+    <string name="title_checkbox_mouse_nav_buttons">\'Back\' and \'Forward\' mouse buttons</string>
+    <string name="summary_checkbox_mouse_nav_buttons">May break right-clicking on some devices</string>
     <string name="title_checkbox_flip_face_buttons">Flip face buttons</string>
-    <string name="summary_checkbox_flip_face_buttons">Switches the face buttons A/B and X/Y for gamepads and the on-screen controls</string>
+    <string name="summary_checkbox_flip_face_buttons">Switches A/B and X/Y for gamepads and on-screen controller</string>
 
-    <string name="category_on_screen_controls_settings">On-screen Controls Settings</string>
-    <string name="title_checkbox_show_onscreen_controls">Show on-screen controls</string>
+    <string name="category_on_screen_controls_settings">On-screen Controller</string>
+    <string name="title_checkbox_show_onscreen_controls">Show on-screen controller</string>
     <string name="summary_checkbox_show_onscreen_controls">Show virtual controller overlay on touchscreen</string>
-    <string name="title_checkbox_vibrate_osc">Enable vibration</string>
-    <string name="summary_checkbox_vibrate_osc">Vibrates your device to emulate rumble for the on-screen controls</string>
+    <string name="title_checkbox_vibrate_osc">Vibration</string>
```

Comment:
Not sure about this at all. How is it different from the emulated vibration if it is still the device that vibrates?